src/Sulu/Bundle/MediaBundle/Resources/js/containers/MediaContainer/stores/CollectionInfoStore.js

```diff
@@ -0,0 +1,91 @@
+// @flow
+import {action, autorun, observable} from 'mobx';
+import {ResourceRequester} from 'sulu-admin-bundle/services';
+import type {BreadcrumbItem, BreadcrumbItems} from '../types';
+
+const COLLECTIONS_RESOURCE_KEY = 'collections';
+
+export default class CollectionInfoStore {
+    @observable loading: boolean = false;
+    @observable breadcrumb: ?observable = [];
+    @observable parentCollectionId: ?observable;
+    disposer: () => void;
+
+    constructor(collectionId: ?string | number, locale: string) {
```

Comment:
Probably but ids and values are typed like this in the whole app. I would leave that (at least for now) to stay consistent and maybe change it later in an own PR

Reply:
But a collection is always a `number`, and therefore that value would also be accepted by anything typed with `string | number`. And it might save one or the other check if we be more clear about that here.